src/classes/Frames.ts

```diff
@@ -0,0 +1,49 @@
+import { ErrorType } from "../types";
+
+export const getTFRFrames = async ({ sku }): Promise<ErrorType | void> => {
+  window.theFittingRoom?.getUserProfile().then(userProfile => {
+    console.log("userProfile: ", userProfile?.vto);
+    console.log("sku: ", sku);
+
+    const getBrandId = () => {
```

Comment:
This function name is incorrect. It is getting the band style id, not the brand id. We should name it accordingly.